exe-common/examples/generate_genesis_data.rs

```diff
@@ -97,10 +97,7 @@ fn main() {
         start_time: SystemTime::UNIX_EPOCH + Duration::from_secs(1548089245),
         slot_duration: Duration::from_millis(20000),
         protocol_magic,
-        fee_policy: fee::LinearFee::new(
-            fee::Milli::integral(43946000),
-            fee::Milli::integral(155381000000),
-        ),
+        fee_policy: fee::LinearFee::new(fee::Milli::new(43, 946), fee::Milli::integral(155381)),
```

Comment:
The parameter are inversed and not correct I believe

Reply:
https://github.com/input-output-hk/rust-cardano/blob/master/cardano/src/fee.rs#L162

Reply:
The constant is the first parameter : https://github.com/input-output-hk/rust-cardano/blob/master/cardano/src/fee.rs#L117-L122